setup.py

```diff
@@ -74,7 +74,7 @@
 install_requires = [
     'Flask-BabelEx>=0.9.3',
     'Flask-Principal>=0.4.0,<0.5.0',
-    'invenio-access>=1.1.0,<1.2.0',
+    'invenio-access>=1.3.0',
```

Comment:
Minor: Shall we put a `<1.4.0`? to be safe?

Reply:
Ok because of https://invenio.readthedocs.io/en/latest/releases/maintenance-policy.html#policy ... >_<